.github/PULL_REQUEST_TEMPLATE/hosted-project.md

```diff
@@ -0,0 +1,22 @@
+---
+name: ✨ New example project
+about: Add an example project to this repository
+---
+
+<!--
+Thank you for your pull request. Please provide a description above and review
+the requirements below.
+
+Contributors guide: https://github.com/fastify/example#contributions
+-->
+
+#### Checklist for a new example
+
+- [ ] the project has a dedicated folder in `kebab-case`
+- [ ] there is not an `index.js`, but a valuable name (usaully the same of the folder)
+- [ ] a test example is included (preferring using `tap`)
```

Comment:
I am not sure if we should express a preference for a test framework.

If we do it should be mentioned in the `External example`, too.

Reply:
The idea is that the example in this repo could be updated by us at some point in time, the external no